src/v/cluster/cloud_metadata/uploader.h

```diff
@@ -27,6 +27,10 @@ namespace raft {
 class group_manager;
 } // namespace raft
 
+namespace storage {
+class api;
+} // namespace storage
```

Comment:
nit: could you include `storage/fwd.h` instead?

Reply:
Done